### ISSUE TITLE
The output shape is empty

### ISSUE THREAD

**Issue by ZhukIvan** [onnx/onnx-coreml#464]
`cml_model = convert(onnx_model)`
Despite the correct short description, the code above gives the output shape of MultiArray is empty.

If you load the model through coremltools, it gives you something like this:

<details>
  <summary>coremltools.models.MLModel(convert(onnx_model))</summary>
    
input {
  name: "image"
  shortDescription: "MultiArray of shape (1, 1, 3, 1280, 720). The first and second dimensions correspond to sequence and batch size, respectively"
  type {
    multiArrayType {
      shape: 3
      shape: 1280
      shape: 720
      dataType: FLOAT32
    }
  }
}
output {
  name: "logits"
  shortDescription: "MultiArray of shape (1, 1, 3, 1280, 720). The first and second dimensions correspond to sequence and batch size, respectively"
  type {
    multiArrayType {
      dataType: FLOAT32
    }
  }
}
    
</details>

If you have 3 channels in the output and convert like this:
`cml_model = convert(onnx_model, image_output_names='logits')`
Conversation would be successful, Xcode will compile it and you will have output like this:
<details>
  <summary>coremltools.models.MLModel(convert(onnx_model, image_output_names='logits'))</summary>
    
input {
  name: "image"
  shortDescription: "MultiArray of shape (1, 1, 3, 1280, 720). The first and second dimensions correspond to sequence and batch size, respectively"
  type {
    multiArrayType {
      shape: 3
      shape: 1280
      shape: 720
      dataType: FLOAT32
    }
  }
}
output {
  name: "logits"
  type {
    imageType {
      width: 720
      height: 1280
      colorSpace: RGB
    }
  }
}
    
</details>

If you make convert like this:
`cml_model = convert(onnx_model, disable_coreml_rank5_mapping=True)`
you will get tensors with 4 dimensions and correct output shape, but also you will get ["pixel_shuffle_reshape" has already been added](https://github.com/onnx/onnx-coreml/issues/446) error in some cases. 
In other cases you will get output:
<details>
  <summary>coremltools.models.MLModel(convert(onnx_model, disable_coreml_rank5_mapping=True))</summary>
    
input {
  name: "image"
  type {
    multiArrayType {
      shape: 1
      shape: 3
      shape: 1280
      shape: 720
      dataType: FLOAT32
    }
  }
}
output {
  name: "logits"
  type {
    multiArrayType {
      shape: 1
      shape: 3
      shape: 1280
      shape: 720
      dataType: FLOAT32
    }
  }
}
     
</details>

Netron also shows the empty output in mlmodel.

<img width="1920" alt="Screen Shot 2019-09-04 at 3 33 28 PM" src="https://user-images.githubusercontent.com/18258728/64255141-754b7900-cf29-11e9-8364-f7ce6a8d6bf7.png">

Here is [onnx](https://mega.nz/#!8RBXXIhZ!W_zPpfB3FKmafEcEoDzNytcSM_tR_RJGrC99AyVcHmk) and [mlmodel](https://mega.nz/#!MMID3YAL!cl9jMJBXmcmOuilbzmwuAtFQQhZdYrGWdDXIRXGeKLw) files for test

**Comment by bhushan23** [comment]
@ZhukIvan can you share ONNX model?

**Comment by ZhukIvan** [comment]
> @ZhukIvan can you share ONNX model?

But I did, there are link under the image for onnx and coreml models. 
Here it is [onnx model](https://mega.nz/#!8RBXXIhZ!W_zPpfB3FKmafEcEoDzNytcSM_tR_RJGrC99AyVcHmk)

**Comment by bhushan23** [comment]
@ZhukIvan i am on tot onnx-coreml and coremltools
I am able to successfully convert with shape information as well.
Could you please try with the latest onnx-coreml and coremltools?

**Comment by ZhukIvan** [comment]
@bhushan23 Could you please send output of `coremltools.models.MLModel(convert(onnx_model))` command and resulted coreml model ?

**Comment by bhushan23** [comment]
@ZhukIvan here is converted model https://drive.google.com/open?id=1F39NfIeCdD1UjbeXbEVK7qIJZ7uJH051

Following is script to convert
```
import onnx
from onnx_coreml import convert

## Load ONNX Model
model = onnx.load('/Volumes/Common/Work/issues/github/name/mt.onnx')

## Convert ONNX Model into CoreML MLModel
c_model = convert(model, disable_coreml_rank5_mapping=True)
c_model.save('converted.mlmodel')
```

Following is stdout

```
WARNING:root:TensorFlow version 1.14.0 detected. Last version known to be fully compatible is 1.13.1 .
WARNING:root:Keras version 2.2.5 detected. Last version known to be fully compatible of Keras is 2.2.4 .
1/2: Converting Node Type Conv
2/2: Converting Node Type Relu
Translation to CoreML spec completed. Now compiling the CoreML model.
Model Compilation done.
```

**Comment by ZhukIvan** [comment]
@bhushan23 That's true, and I wrote exactly about this. Without flag you get empty shape, with flag you get good shape, but in some cases you will get [pixel shuffle reshape error](https://github.com/onnx/onnx-coreml/issues/446). Please, try to convert with `disable_coreml_rank5_mapping =True` this [onnx model](https://mega.nz/#!NYIQFQDQ!xroJ2stsuy3_fT4Tq4oYXa9LFklmoVfc_lEm7XUgDr0).
Thank you!

**Comment by bhushan23** [comment]
@ZhukIvan I am indeed converting with `disable_coreml_rank5_mapping=True` flag.
please take a look at script provided in above google drive link ([convert.py](https://drive.google.com/open?id=1XXRDs5QDy_OWKdQ8HiN0kv7oO0pFv_vR))

We have one more way to make a progress here, could you please set DEBUG=True in converter.py (https://github.com/onnx/onnx-coreml/blob/b3154187cd0549eb55b7b19a61586fe3ade14b11/onnx_coreml/converter.py#L35) and try converting again? this will dump neural network specification as well.

**Comment by ZhukIvan** [comment]
@bhushan23 You was right, I've converted model with pytorch 1.1, but after upgrading to 1.2 version error remains. Currently I have this versions:
```
onnx==1.5.0
onnx-coreml==1.0b2
torch==1.2.0
coremltools==3.0b6
```
Here is pytorch [model](https://mega.nz/#!0UBWAYJK!I731_Fyjpz_Uhl9z1DHVT6FVMixdARzSqL9dCA6Ku0E) and here is conversion code :
```python
dummy_in = torch.autograd.Variable(torch.randn(1, 3, 640, 480)).cuda()
torch.onnx.export(model, dummy_in, 'model.onnx', input_names=['image'],  
        output_names=['logits'], verbose=True)
```

**Comment by bhushan23** [comment]
@ZhukIvan This should be fixed with https://github.com/onnx/onnx-coreml/pull/481
